cypress/e2e/item.cy.js

```diff
@@ -33,6 +33,9 @@ describe('Create and Delete item positive test Suite', () => {
     cy.visit(page.profile, { failOnStatusCode: false });
     ProfilePage.deleteItemButton(uniqueItemName).click();
     ProfilePage.ConfirmDeleteItemButton(uniqueItemName).click();
-    ProfilePage.itemCard(uniqueItemName).should('not.exist');
+    cy.reload();
+    ProfilePage.itemCard(uniqueItemName).should('not.exist', {
+      timeout: 10000,
```

Comment:
why is the pr for the `add-item` form changing the test that deletes an item? did something break?

we should not need to reload because react is taking care of hiding the deleted items as they get removed presumably